static/js/click_handlers.js

```diff
@@ -988,13 +993,16 @@ export function initialize() {
         }
 
         if (compose_state.composing()) {
+            const $closest_links = $(e.target).closest("a");
             if (
-                $(e.target).closest("a").length > 0 ||
+                ($closest_links.length > 0 && $closest_links[0].role !== "menuitem") ||
```

Comment:
Narrowing the scope for this codepath to not have dropdown menu clicks end up here.

Reply:
I would expect this to belong in the `!$(e.target).closest(".overlay").length` class of block just below.

You could probably submit a version of this as a prep commit -- I think basically we just want "clicks inside a `dropdown_list_widget`" never trigger the below logic, just like clicks inside a popover do.

Reply:
Can you clarify what you mean by belonging in the block below?

`$(e.target).closest("a").length > 0` is true for dropdown clicks, and we need some way to not get inside this if block.

Reply:
Put a prep commit for this in #24133